mooncake-transfer-engine/src/transfer_metadata_plugin.cpp

```diff
@@ -1086,11 +1090,13 @@ uint16_t findAvailableTcpPort(int &sockfd) {
             continue;
         }
 
-        int on = 1;
-        if (setsockopt(sockfd, SOL_SOCKET, SO_REUSEADDR, &on, sizeof(on))) {
-            close(sockfd);
-            sockfd = -1;
-            continue;
+        if (getenv("MC_ENABLE_REUSEADDR")) {
+            int on = 1;
+            if (setsockopt(sockfd, SOL_SOCKET, SO_REUSEADDR, &on, sizeof(on))) {
+                close(sockfd);
+                sockfd = -1;
+                continue;
+            }
         }
```

Comment:
we'd better remove these lines? multiple TE reuse the same port is not expected, since only one TE could accept the connection from client.